connaisseur/validate.py

```diff
@@ -225,3 +201,38 @@ def search_image_targets_for_tag(trust_data: dict, image: Image):
 
     base64_digest = trust_data[image_tag]["hashes"]["sha256"]
     return base64.b64decode(base64_digest).hex()
+
+
+def update_with_delegation_trust_data(trust_data, delegations, key_store, host, image):
```

Comment:
If this function is only used in this file, you may want to make it private (prepend with `_`)